src/js_command_match/mod.rs

```diff
@@ -32,39 +32,36 @@ pub enum JsCmd {
     Send(self::transactions::send::SendArgs),
     TxStatus(self::transactions::status::TxStatusArgs),
 
-    #[clap(alias("validator-stake"))]
```

Comment:
The `stake` command has `validator-stake` in JS version: https://github.com/near/near-cli/blob/31567276536c35f00c4a1a08a362b51cdf7fbb27/commands/validators/stake.js#L10-L11

Let's add this alias back to Stack command